src/test/java/org/sagebionetworks/bridge/workers/dynamodb/streams/StreamsUtilsTest.java

```diff
@@ -0,0 +1,5 @@
+package org.sagebionetworks.bridge.workers.dynamodb.streams;
+
+public class StreamsUtilsTest {
```

Comment:
Empty test file?

It might make it easier to test StreamsUtils if it were non-static and there were a non-static wrapper around DynamoUtils.getExistingTables() so you can mock it.